storage/block/types.go

```diff
@@ -317,12 +315,24 @@ type Options interface {
 	// ClockOptions returns the clock options
 	ClockOptions() clock.Options
 
-	// SetDatabaseBlockAllocSize sets the databaseBlockAllocSize
+	// SetInstrumentOptions sets the instrument options
+	SetInstrumentOptions(value instrument.Options) Options
+
+	// InstrumentOptions returns the instrument options
+	InstrumentOptions() instrument.Options
+
+	// SetDatabaseBlockAllocSize sets the database block alloc size
 	SetDatabaseBlockAllocSize(value int) Options
 
-	// DatabaseBlockAllocSize returns the databaseBlockAllocSize
+	// DatabaseBlockAllocSize returns the database block alloc size
 	DatabaseBlockAllocSize() int
 
+	// SetWiredList sets the database block wired list
+	SetWiredList(value *WiredList) Options
```

Comment:
Thankfully it's relatively simple and is safe to use across different layers, i.e. we don't need to mock it.  Hence why I'm currently inclined to leave it as a pointer type.